mat64/qr.go

```diff
@@ -209,21 +212,7 @@ func (v *Vector) SolveQRVec(qr *QR, trans bool, b *Vector) error {
 	} else {
 		v.reuseAs(c)
 	}
-	m := vecAsDense(v)
-	bm := vecAsDense(b)
+	m := v.asDense()
+	bm := b.asDense()
```

Comment:
Considering moving these into the call line as `return v.asDense().SolveQR(qr, trans, b.asDense())`.